building-locally.md

```diff
@@ -0,0 +1,137 @@
+
+1. Clone this repository.
+
+   `git clone https://github.com/ibm-cloud-architecture/refarch-cloudnative-micro-customer.git`
+   
+   `cd refarch-cloudnative-micro-customer/`
+
+2. Checkout MicroProfile branch.
+
+   `git checkout microprofile`
+
+3. Run this command. This command builds the project and installs it.
+
+   `mvn install`
+   
+   If this runs successfully, you will be able to see the below messages.
+
+```
+[INFO] --- maven-failsafe-plugin:2.18.1:verify (verify-results) @ customer ---
+[INFO] Failsafe report directory: /Users/user@ibm.com/Desktop/BlueCompute/refarch-cloudnative-micro-customer/target/test-reports/it
+[INFO]
+[INFO] --- maven-install-plugin:2.4:install (default-install) @ customer ---
+[INFO] Installing /Users/user@ibm.com/Desktop/BlueCompute/refarch-cloudnative-micro-customer/target/customer-1.0-SNAPSHOT.war to /Users/user@ibm.com/.m2/repository/projects/customer/1.0-SNAPSHOT/customer-1.0-SNAPSHOT.war
+[INFO] Installing /Users/user@ibm.com/Desktop/BlueCompute/refarch-cloudnative-micro-customer/pom.xml to /Users/user@ibm.com/.m2/repository/projects/customer/1.0-SNAPSHOT/customer-1.0-SNAPSHOT.pom
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 15.788 s
+[INFO] Finished at: 2018-07-23T16:38:19-05:00
+[INFO] Final Memory: 26M/266M
+[INFO] ------------------------------------------------------------------------
+```
+
+4. The Auth service will also be running. A JWT is required to use the Customer Service Locally.
+https://github.com/ibm-cloud-architecture/refarch-cloudnative-auth/tree/microprofile
+
+
+## Setting up Cloudant
+
+To set up the Cloudant database locally, we are running it as a docker container. You need [Docker](https://www.docker.com/) as a prerequisite.
+To run cloudant on docker locally, run the commands below.
+
+1.
+```
+docker pull ibmcom/cloudant-developer
+```
+
+
+2. In this case, you will need to set this property in microprofile-config.properties
+https://github.com/ibm-cloud-architecture/refarch-cloudnative-micro-customer/blob/microprofile/src/main/resources/META-INF/microprofile-config.properties
+
+```
+application.rest.client.CloudantClientService/mp-rest/url=http://localhost:8080
+```
+
+## Setting up Zipkin
+
+This is an optional step.
+
+In our sample application, we used Zipkin as our distributed tracing system.
+
+If you want to access the traces for inventory service, run Zipkin as a docker container locally. You can find the instructions and more details [here](https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/blob/microprofile/Zipkin/README.md)
+
+### Running the app and stopping it
+
+1. Set the JDBC URL before you start your application. The host and port depends on the service you use.
```

Comment:
I think we can remove the line - Set the JDBC URL here. Also, please mention that for this service to run individually, auth and keystore are mandatory and they should be up and running. Link those instructions as well.